src/main/java/com/dnastack/wes/storage/GcpBlobStorageClient.java

```diff
@@ -118,12 +118,23 @@ public void readBytes(OutputStream outputStream, String blobUri, HttpRange httpR
     @Override
     public boolean isFile(String filePath) {
         try {
-            return client.get(GcpStorageUtils.blobIdFromGsUrl(filePath)).exists();
+            BlobId blob = GcpStorageUtils.blobIdFromGsUrl(filePath);
```

Comment:
Let's replace calls for `GcpStorageUtils.blobIdFromGsUrl(filePath)` with:

```java
private BlobId getBlobIdFrom(String filePath) {
  return GcpStorageUtils.blobIdFromGsUrl(filePath);
}
```

Reply:
This seems a bit redundant